build.rs

```diff
@@ -20,6 +20,8 @@ use std::{
 
 #[cfg(target_os = "windows")]
 const EXPECTED_JVM_FILENAME: &str = "jvm.dll";
+#[cfg(target_os = "freebsd")]
+const EXPECTED_JVM_FILENAME: &str = "libjvm.so";
 #[cfg(target_os = "linux")]
 const EXPECTED_JVM_FILENAME: &str = "libjvm.so";
```

Comment:
I suppose it would be better to use `cfg(any(...))` because the library name is the same:
```suggestion
#[cfg(any(target_os = "linux", target_os = "freebsd"))]
const EXPECTED_JVM_FILENAME: &str = "libjvm.so";
```